python/dgl/graphbolt/minibatch.py

```diff
@@ -474,6 +474,35 @@ def apply_to(x, device):
         return self
 
 
+    def to_pyg_adapter(self, device):
+        from torch_geometric.data import Data
+
+        def construct_edge_index(subgraph, device):
+            csc_matrix = subgraph.sampled_csc
+            indices = csc_matrix.indices.to(device)
+            indptr = csc_matrix.indptr.to(device)
+            row_indices = torch.arange(
+                len(indptr) - 1, device=device
+            ).repeat_interleave(indptr.diff())
+            col_indices = indices
+            return torch.stack([row_indices, col_indices], dim=0)
+
+        all_edge_indices = []
```

Comment:
I'm little confused that why the name is `indices` but not `index`?

Reply:
Because it is to represent multiple index values